module/gurps.js

```diff
@@ -2581,15 +2562,18 @@ if (!globalThis.GURPS) {
       GurpsWiring.hookupAllEvents(html)
     })
 
+    // TODO: Move to the combat module?  We could have a method in combat that allows other modules to request hooks into the Combat system.
```

Comment:
agreed on the comment here. we shouldn't really be using hooks for things we can use subclasses for.